draft-packages/select/KaizenDraft/Select/Select.tsx

```diff
@@ -14,7 +14,71 @@ const styles = require("./styles.react.scss")
 
 export { ValueType } from "react-select"
 
-export const Select = (props: ReactSelectProps) => {
+export type SelectProps = {
+  /**
+   * The secondary variant is a more subdued variant that takes up as little space as possible
+   * `variant="secondary" reversed="false" is not implemented and will throw a "not implemented" error
+   * @default "default"
+   */
+  variant?: VariantType
+
+  /**
+   * Use a reversed colour scheme
+   * `variant="default" reversed="true" is not implemented and will throw a "not implemented" error
+   * @default false
+   */
+  reversed?: boolean
+
+  /**
+   * Whether the "control" (the button you click to open the menu) width fills the
+   * container or is as wide as the selected option text. Note that the control text
+   * will ellipsize if it is wider than the parent container.
+   * `variant="default" selectControlWidth="containSelection"` is not implemented and
+   * will throw a "not implemented" error
+   * @default "containSelection" if variant is "secondary", otherwise "fillContainer"
+   */
+  selectControlWidth?: SelectControlWidthType
+}
+
+export type VariantType = "default" | "secondary"
+export type SelectControlWidthType = "fillContainer" | "containSelection"
```

Comment:
Since the options here are binary, I'd prefer to use `fullWidth?: boolean`. 
fullWidth = true -> fillContainer
fullWidth = false -> containSelection

https://github.com/cultureamp/kaizen-design-system/blob/master/packages/component-library/components/Button/Button.tsx#L9

Reply:
We arrived at this in this thread: https://cultureamp.slack.com/archives/GF02FQA30/p1592366131161600

We didn't consider "full width" true/false, but I guess we were trying to make it as self documentating as possible.
There's a potential third option: "widthOfTheWidestOption", so making it non-binary gives flexibility to add other options later. Happy to delay this PR until @didoesdigital is back.